polyfactory/factories/base.py

```diff
@@ -617,6 +634,67 @@ def get_field_value(  # noqa: C901, PLR0911, PLR0912
             msg,
         )
 
+    @classmethod
+    def get_field_value_coverage(  # noqa: C901
+        cls,
+        field_meta: FieldMeta,
+        field_build_parameters: Any | None = None,
+    ) -> abc.Iterable[Any]:
```

Comment:
@sobolevn, should the return types be `typing.Iterable[Any]` here? Are there any differences in using `abc.<SomeType>` vs `typing.<SomeType>`?

Reply:
With `__future__.annotations` this shouldn't matter. Without that it would be a runtime error on 3.8.